hazelcast/src/main/java/com/hazelcast/query/impl/ReflectionHelper.java

```diff
@@ -169,6 +170,16 @@ public static Comparable extractValue(Object object, String attributeName) throw
         return (Comparable) createGetter(object, attributeName).getValue(object);
     }
 
+    public static <T> T invokeMethod(Object object, String methodName) throws RuntimeException {
```

Comment:
Looks like this `ReflectionHelper ` is targeted to use with queries. Maybe you can create a global utility class for methods like `invokeMethod`.